exampleSite/config.toml

```diff
@@ -46,6 +46,9 @@ disqusShortname = "yourdiscussshortname"
     # Series see also post count
     maxSeeAlsoItems = 5
 
+    # Use Twemoji
+    useTwemoji = true
```

Comment:
In the [gohugo docs](https://gohugo.io/getting-started/configuration/#all-configuration-settings), they use the `enableEmoji` config parameter to enable Emoji emoticons support for page content, so I suggest using `enableTwemoji` instead.